index.js

```diff
@@ -3,34 +3,27 @@ var path = require('path');
 var pathExists = require('path-exists');
 var globby = require('globby');
 
+// A resolve that does not accept variadic arguments so it can be used with map.
+function resolvePath(x) {
+	return path.resolve(x);
+}
```

Comment:
Just put it inline. Will look better when I eventually move this to arrow functions.